### PR TITLE
systemd-delta: add page

### PR DESCRIPTION
**Pull request by acuteenvy**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/linux/systemd-delta.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Show only files of specific types (comma-separated list):
+
+`systemd-delta --type {{masked|equivalent|redirected|overridden|extended|unchanged}}`
+
+- Show only files whose path starts with the specified prefix (Note: a prefix is a directory containing subdirectories with systemd configuration files):
+
+`systemd-delta {{/etc|/run|/usr/lib|...}}`
+
+- Further restrict the search path by adding a suffix (the prefix is optional):
```
Comment (kbdharun) [inline pages/linux/systemd-delta.md:18]:
```suggestion
- Further, restrict the search path by adding a suffix (the prefix is optional):
```
Minor nitpick

**Review by kbdharun — APPROVED** [review]

**Comment by Waples** [comment]
Approved and implemented the ',' suggestion. LGTM further more.

File: pages/linux/systemd-delta.md (hunk 1 above)
Comment (kbdharun) [reply pages/linux/systemd-delta.md]: I generally use proofreading tools like Grammarly, etc  to validate the PRs in this the reason for the suggestion is

> An introductory word (Unfortunately), phrase (Before the party), or clause (If you can join us) sets the stage for the sentence. Introductory clauses and long phrases must always be followed by a comma, while single words and short phrases may or may not. Use a comma when the phrase is a verbal phrase (Running away), a non-essential appositive, or an absolute phrase (The audience waiting). Also use a comma when a pause is necessary or the lack of a comma will cause confusion.

IMO, both with or without the comma works that's why I suggested it as nitpick while approving the PR.

Comment (acuteenvy) [reply pages/linux/systemd-delta.md]: In this case "further" is not an introduction, it describes "restrict". That's why I don't think there should be a comma.

Comment (kbdharun) [reply pages/linux/systemd-delta.md]: > In this case "further" is not an introduction, it describes "restrict". That's why I don't think there should be a comma.

Agreed, feel free to revert the change.

**Review by kbdharun — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**